apps/web/app/(app)/environments/[environmentId]/product/(setup)/website-connection/loading.tsx

```diff
@@ -1,49 +1,41 @@
 "use client";
 
-import { BrushIcon, KeyIcon, LanguagesIcon, ListChecksIcon, TagIcon, UsersIcon } from "lucide-react";
-import { usePathname } from "next/navigation";
+import { ProductConfigNavigation } from "@/app/(app)/environments/[environmentId]/product/components/ProductConfigNavigation";
+import { SettingsCard } from "@/app/(app)/environments/[environmentId]/settings/components/SettingsCard";
 import { cn } from "@formbricks/lib/cn";
 import { PageContentWrapper } from "@formbricks/ui/components/PageContentWrapper";
 import { PageHeader } from "@formbricks/ui/components/PageHeader";
 
 const LoadingCard = ({ title, description, skeletonLines }) => {
```

Comment:
We're having this LoadingCard in each loading.tsx of the settings -> Feels like it should be its own component.